src/utils.js

```diff
@@ -239,3 +239,22 @@ export function scrollToBottom(): Promise<void> {
   window.scrollTo(0, window.document.body.scrollHeight);
   return Promise.resolve();
 }
+
+export function sortHistoryEntryPermissions(
+  entry: ResourceHistoryEntry
+): ResourceHistoryEntry {
+  if (entry.action === "delete") {
+    return entry;
+  }
+  const { permissions } = entry.target;
```

Comment:
Rather than checking explicitly for `delete`, should we just check for the absence of `permissions`?